Dockerfile

```diff
@@ -0,0 +1,44 @@
+# syntax=docker.io/docker/dockerfile:1.7-labs
+ARG DOTNET_VERSION=9.0.101
+ARG OS=linux
+ARG ARCH=x64
+
+FROM ubuntu:24.10 AS dependencies
+ARG DOTNET_VERSION
+ARG ARCH
+ARG OS
+
+RUN apt-get update -y \
+    && apt-get install -y libicu-dev curl
+
+RUN curl -L https://dotnet.microsoft.com/download/dotnet/scripts/v1/dotnet-install.sh -o dotnet-install.sh && \
+    chmod +x dotnet-install.sh && \
+    ./dotnet-install.sh --version ${DOTNET_VERSION} --architecture ${ARCH} --os ${OS}
+
+ENV PATH $PATH:/root/.dotnet
+
+FROM dependencies AS publish
+ARG OS
+ARG ARCH
+
+WORKDIR /bin/modernuo
+
+COPY . .
+
+RUN ./publish.sh release ${OS} ${ARCH}
+
+FROM dependencies AS deploy
+ARG DOTNET_VERSION
+ARG ARCH
+
+ENV DOTNET_ROOT=/root/.dotnet
```

Comment:
updating the `PATH` in the `dependencies` stage above doesn't enable the `ModernUO` binary to find that installation

instead, we have to set `DOTNET_ROOT` before we run the binary